components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -467,8 +434,8 @@ protected void findPreviousAuthenticatedSession(HttpServletRequest request,
                         try {
                             effectiveSequence = (SequenceConfig) previousAuthenticatedSeq.clone();
                         } catch (CloneNotSupportedException e) {
-                            throw new FrameworkException("Exception when trying to clone the Previous Authentication " +
-                                    "Sequence object of SP:" + appName, e);
+                            throw new FrameworkException("Exception when trying to clone the Previous Authentication "
+                                    + "Sequence object of SP:" + appName, e);
```

Comment:
There seems to be a formatting issue near + mark

Reply:
Looks like it has a trailing space so looks fine. Still I think it would be better to have a space after 'Sequence object of SP:' adding more clarity to the log